src/cityreader/cityreader.py

```diff
@@ -15,12 +22,19 @@
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
 cities = []
-
+import csv
 def cityreader(cities=[]):
   # TODO Implement the functionality to read from the 'cities.csv' file
   # For each city record, create a new City instance and add it to the 
   # `cities` list
-    
+    with open('cities.csv') as csvfile:
+      readCSV = csv.reader(csvfile)
+      firstline = True
+      for row in readCSV:
+            if firstline:
+                  firstline = False
+                  continue
```

Comment:
As we spoke about, you could used `next()` to skip the header in csv or spliced the first index from the readCSV. However, given its your first week in python, I think this is an innovative solution. Great work.